src/components/Responsive/Visible/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import intersection from 'lodash/intersection';
+
+import Context from '../Context';
+
+const Visible = ({ children, breakpoints, ...passedProps }) => {
+  const getIsVisible = useCallback(
+    responsiveContext => !!intersection(breakpoints, responsiveContext).length,
```

Comment:
I tried using the `!!` over in tk world and received a lint exception for it (Redundant double negation.eslint(no-extra-boolean-cast). Is it ok to use this over here?

Reply:
:man_shrugging: Ya I didn't understand why it would warn me (like it was a bad idea to use it). Anyhow, approved the request.